kolibri/plugins/setup_wizard/assets/src/views/LoadingTaskPage.vue

```diff
@@ -124,9 +130,10 @@
       isSoud() {
         return this.queue === SoudQueue;
       },
-      isImportingSoud() {
-        return this.wizardService.state.context.lodImportOrJoin === LodTypePresets.IMPORT;
-      },
+      // To Do: role this back when multiple user syncing is working in the upcoming release!
```

Comment:
See comment from @nucleogenesis below. Its also good to clean out any commented out code.

Reply:
Hi @akolson , this is a temporary fix we shall need to reinstate it easily in future.